extensions/v2ex/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "v2ex",
+  "title": "v2ex",
+  "description": "hot list of v2ex",
```

```suggestion
  "description": "Read the hot list of v2ex",
```

(The description is displayed in the store, so it should at least be a sentence ;). Feel free to add more if you want.)